app/models/operatingsystem.rb

```diff
@@ -32,7 +32,6 @@ class Operatingsystem < ApplicationRecord
   include ParameterSearch
 
   attr_name :to_label
-  validates :minor, :numericality => {:greater_than_or_equal_to => 0}, :allow_nil => true, :allow_blank => true
```

Comment:
Can we use regex (`/^(\d+\.)*\d+$/`) to validate `minor`?

Reply:
Sure.